pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -79,6 +79,21 @@ type CiliumBGPNeighbor struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4294967295
 	PeerASN int `json:"peerASN"`
+	// ConnectRetryTime defines the initial value for the BGP ConnectRetryTimer (RFC 4271, Section 8).
+	// The default value for the ConnectRetryTime (if empty or zero) is 120 seconds.
+	//
+	// +kubebuilder:validation:Optional
+	ConnectRetryTime metav1.Duration `json:"connectRetryTime,omitempty"`
+	// HoldTime defines the initial value for the BGP HoldTimer (RFC 4271, Section 4.2).
+	// The default value for the HoldTime (if empty or zero) is 90 seconds.
+	//
+	// +kubebuilder:validation:Optional
+	HoldTime metav1.Duration `json:"holdTime,omitempty"`
+	// KeepaliveTime defines the initial value for the BGP KeepaliveTimer (RFC 4271, Section 8).
+	// The default value for the KeepaliveTime (if empty or zero) is 1/3 of the HoldTime.
+	//
+	// +kubebuilder:validation:Optional
+	KeepAliveTime metav1.Duration `json:"keepAliveTime,omitempty"`
```

Comment:
Also, it doesn't make sense to have KeepAlive time larger than hold time. Can we make any validation for that?

Reply:
Added a runtime check for this. Once CEL-based [CRD Validation Rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) are supported in the Cilium repo, I will enhance this with additional schema validations.